books/RayTracingTheNextWeek.html

```diff
@@ -1097,47 +1118,64 @@
         bool front_face;
         ...
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [hit-record-uv]: <kbd>[hittable.h]</kbd> Adding U,V coordinates to the `hit_record`]
+    [Listing [hit-record-uv]: <kbd>[hittable.h]</kbd> Adding $u,v$ coordinates to the `hit_record`]
 
-We will also need to compute $(u,v)$ texture coordinates for hittables.
+We will also need to compute $(u,v)$ texture coordinates for a given point on each type of
+`hittable`.
 
 
 Solid Textures: A Checker Texture
 ----------------------------------
-<div class='together'>
-We can create a checker texture by noting that the sign of sine and cosine just alternates in a
-regular way, and if we multiply trig functions in all three dimensions, the sign of that product
-forms a 3D checker pattern.
+The next simplest texture map type is the solid texture, which depends only on the position of each
```

Comment:
"The simplest texture map type is the solid texture..." ?

Reply:
I was asserting that if the solid color is the simplest texture map, the spatial/solid texture is the next simplest. So technically correct, but it struck me as awkward too. Rethinking...